### DISCUSSION TITLE
Export motion photos as GIFs

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#920] [Enhancements]
Provide an option to share / export the video content within a motion photo as a GIF.

[video_trimmer](https://pub.dev/packages/video_trimmer) accepts  `customVideoFormat` as a parameter, that can be used to convert the video stream fetched from the motion photos plug-in.

**Comment by laurenspriem** [comment]
I think in general for any video it would be cool to be able to export it to a GIF using ente, not just for motion photos.
